modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/ObjectEntryFolderResourceImpl.java

```diff
@@ -65,6 +87,67 @@ public Page<ObjectEntryFolder> getAssetLibraryObjectEntryFoldersPage(
 			assetLibraryId, pagination);
 	}
 
+	@Override
+	public ObjectEntryFolder getObjectEntryFolder(Long objectEntryFolderId)
+		throws Exception {
+
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-17564")) {
+			throw new UnsupportedOperationException();
+		}
+
+		return _toObjectEntryFolder(
+			_objectEntryFolderService.getObjectEntryFolder(
+				objectEntryFolderId));
+	}
+
+	@Override
+	public ObjectEntryFolder patchObjectEntryFolder(
+			Long objectEntryFolderId, ObjectEntryFolder objectEntryFolder)
+		throws Exception {
+
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-17564")) {
+			throw new UnsupportedOperationException();
+		}
+
+		com.liferay.object.model.ObjectEntryFolder persistedObjectEntryFolder =
+			_objectEntryFolderService.getObjectEntryFolder(objectEntryFolderId);
+
+		Long parentObjectEntryFolderId =
+			objectEntryFolder.getParentObjectEntryFolderId();
+
+		if (parentObjectEntryFolderId == null) {
+			parentObjectEntryFolderId =
+				persistedObjectEntryFolder.getParentObjectEntryFolderId();
+		}
+
+		String name = objectEntryFolder.getName();
+
+		if (name == null) {
```

Comment:
String null validation is usually done using the Validator.isNotNull not the regular null check

Reply:
this is made on purpose. I want to know if the value is null, and Validator.isNull on strings takes the blank as null, and "" is a valid value (wrong but valid)